template-sample-app/template/src/components/views/stores/StoresList.js

```diff
@@ -57,9 +57,9 @@ export default function StoresList() {
         id='storesByRevenue'
         title='Stores by revenue'
         dataSource={SOURCE_ID}
-        formatter={numberFormatter}
+        yAxisFormatter={numberFormatter}
```

Comment:
Not happy with this change. It's not the yAxis, it's the value of the tooltip

Reply:
The tooltip is showing the Y axis values...

Reply:
yes @alasarr, this formatter is used in both, tooltip and yAxis

Reply:
tooltipFormatter ?

Reply:
Agree with Iván. Let's keep it (and adjust the lib accordingly)